book/src/getting-started/5-cmake-integration.md

```diff
@@ -77,17 +77,12 @@ In the end, your `Cargo.toml` should look similar to this.
 
 [dependencies]
 cxx = "1.0.95"
-cxx-qt = "0.6"
-cxx-qt-lib = "0.6"
+cxx-qt = "0.7"
+cxx-qt-lib = { version="0.7", features = ["qt_full"] }
```

Comment:
normally i update all the version at the same time when doing a release. 

And there are two options around that, you either update the version numbers as you tag and do the release, or you pre-emptively update them at the start of the development cycle just after a release. So far we have been updating them when we do a release